### PR TITLE
Localize jp

### PR DESCRIPTION
**Pull request by hachi8833**
Retry of  #154

### CODE REVIEW

File: jp/examples/section_1/comment.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Comments
```
Comment (Delta456) [inline jp/examples/section_1/comment.md]: Please translate this section

**Review by Delta456 — CHANGES_REQUESTED** [review]
Please translate that section

**Comment by Delta456** [comment]
Also see #158

File: jp/examples/section_1/comment.md (hunk 1 above)
Comment (hachi8833) [reply jp/examples/section_1/comment.md]: Translated

File: jp/examples/section_1/hello_world.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+---
+version: 1.0.0
+example_title: hello world
```
Comment (Delta456) [inline jp/examples/section_1/hello_world.md:3]: Translate this as well

**Review by Delta456 — CHANGES_REQUESTED** [review]

**Pull request merged**